src/animad-keepers-view.html

```diff
@@ -23,7 +24,17 @@
       }
     </style>
 
-    <div class="card">
+    <app-route route="{{route}}" pattern="/:page" data="{{routeData}}" tail="{{subroute}}">
```

Comment:
Wie machst du denn hier die Interaktion mit der Adressleiste des Browsers. Müsste da nicht irgendwo noch ein `<app-location>` sein? Ich hab noch nicht genau gecheckt, wie das funktioniert :)

Reply:
Siehe Link https://www.polymer-project.org/blog/routing ("... While you’ll have many <app-route> elements in your app, there’s only one URL bar. The URL is global. So we’ve got an element whose single responsibility is connecting the URL to your app....")

Es gibt nur eine URL und damit nur ein app-location. Das Routing wird weitergeleitet. In animad-keepers-view ist dann meine "route" die "subroute" aus animad-app.

Reply:
Ah - okay.